source/components/StyledArticleTagsSelector/SearchForm.js

```diff
@@ -0,0 +1,153 @@
+import debounce from 'lodash/debounce';
+import noop from 'lodash/noop';
```

Comment:
I never understood why anyone would use a third party library for something so simple. What's wrong with `() => {}`?

Reply:
😂 made the same comment. Maybe we should remove

Reply:
Nothing, just less to type. and lodash's already there - and it's the same thing:
https://github.com/lodash/lodash/blob/3.0.1-npm-packages/lodash.noop/index.js#L22-24